packages/ui/src/organisms/OutOfStock/OutOfStock.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import type { ReactNode, FormHTMLAttributes } from 'react'
+
+import Form from '../../molecules/Form'
+
+type OutOfStockBaseProps = {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+
+  /**
+   * Children for Out of Stock components.
+   */
+  children: string | ReactNode
+}
+
+export type OutOfStockProps = OutOfStockBaseProps & {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
```

Comment:
Do we need to redefine this here? Considering it's merging with `OutOfStockBaseProps` that already has it.